laghos.cpp

```diff
@@ -1293,6 +1293,7 @@ int main(int argc, char *argv[])
                     timeLoopTimer.Start();
 
                     basis->ProjectFOMtoROM(S, romS);
+                    cout << myid << ": initial romS norm " << romS.Norml2() << endl;
```

Comment:
How about doing `cout` only `if (myid == 0)`? Also, it would be clearer to output the window index in this line. That is, rather than `initial romS norm` write `window" << romOptions.window << " initial romS norm "`.

Reply:
@dylan-copeland I agree. Let's modify it this way.